src/schema/objects/common_principles.yaml

```diff
@@ -46,6 +46,9 @@ dataset:
   description: |
     A set of neuroimaging and behavioral data acquired for a purpose of a particular study.
     A dataset consists of data acquired from one or more subjects, possibly from multiple sessions.
+derivative:
```

Comment:
wonder if this should be not part of a separate PR

Reply:
Yeah. It's a bit hidden from potential commenters, here.

Reply:
OK will move this out
better change log in the end too